evaluate_arima.py

```diff
@@ -0,0 +1,57 @@
+import warnings
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.arima_model import ARIMA
+from sklearn.metrics import mean_squared_error
```

Comment:
Generally, imports should be in alphabetical order, if you install pre-commit hooks (instructions in README) it should alert you to linting errors.